src/wings_palette.erl

```diff
@@ -98,11 +98,6 @@ forward_event({apply, ReturnSt, Fun}, Window, St0) ->
 	false ->
 	    Fun(St0)
     end;
-forward_event({new_state, #st{}=St}=_, _Window, _) ->
-    %% wings_dialog functions sends {new_state, St} to this window,
-    %% so send it to geom
-    wings_wm:send(geom, {new_state,St}),
-    keep;
```

Comment:
No, this was almost correct :-)
And a real performance bug, we don't want to forward that message to the gui process.

Suggestion:
```
forward_event({new_state, #st{}}=_, _Window, _) ->
    keep;
```